docs/lib/graphqlapi/fragments/flutter/advanced-workflows/40_multiple.md

```diff
@@ -0,0 +1,30 @@
+```dart
+try{
+    String graphQLDocument = '''query get(\$postId: ID!, \$todoId: ID!) {
+          getPost(id: \$postId) {
+            id
+            title
+            rating
+          }
+          getTodo(id: \$todoId) {
+            id
+            name
+          }
+        }''';
+
+    var operation = await Amplify.API
+        .query<String>(request: GraphQLRequest(
+        document: graphQLDocument,
+        variables: {
+            "postId" : "[POST_ID]", 
+            "todoId" : "[TODO_ID]"
+        });
```

Comment:
Same, parenthesis are not balanced and the indentation looks weird. Can we run these samples in the app and also format them with dart formatter?

Reply:
Example updated, issue with copy paste template having that off